src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+                    }
+                    LocalDate d = LocalDate.parse(arr[1].replaceFirst("at", "").stripLeading());
+                    Events curr = new Events(taskName, d);
+                    list.add(curr);
+                    System.out.println(topBound + "Got it! I've added this task:\n" + "  " + curr
+                            + "\nNow you have " + list.noOfTasks() + " tasks in the list." + bottomBound);
+                } catch (DukeException e) {
+                    // command came without a description
+                    System.out.println(topBound + e.getMessage() + bottomBound);
+                } catch (ArrayIndexOutOfBoundsException | DateTimeParseException e) {
+                    // command came without proper date time format
+                    System.out.println(topBound
+                            + "Uh Oh! event must include '/at YYYY-MM-DD'.\nTry again!" + bottomBound);
+                }
+            } else if (arr[0].startsWith("delete")) {
+                try {
+                    String str = arr[0].replaceFirst("delete", "").stripLeading();
+                    int x = Integer.parseInt(str) - 1;
+                    Task curr = list.get(x);
+                    list.delete(x);
+                    System.out.println(topBound + "Got it! I've removed this task:\n" + " " + curr + "\n" + "You have "
+                            + list.noOfTasks() + " tasks left in the list." + bottomBound);
+                } catch (IndexOutOfBoundsException e) {
+                    // task number x does not exist
+                    System.out.println(topBound + "Uh Oh! Task does not exist and cannot be deleted!" + bottomBound);
+                } catch (NumberFormatException e) {
+                    // parameter for delete is not an Integer
+                    System.out.println(topBound
+                            + "Uh Oh! delete must be followed by an Integer, ie. delete 1.\nTry again!" + bottomBound);
+                }
+            } else if (arr[0].startsWith("show")) {
+                try {
+                    LocalDate day = LocalDate.parse(arr[0].replaceFirst("show", "").stripLeading());
+                    System.out.println(topBound + "Here are your tasks on " +
+                            day.format(DateTimeFormatter.ofPattern("MMM dd yyyy")) + ":");
+                    list.showTaskOnDay(day);
+                    System.out.println(bound);
+                } catch (DateTimeParseException e) {
+                    System.out.println(topBound
+                            + "Uh Oh! show must be followed by a date in YYYY-MM-DD format.\nTry again!" + bottomBound);
+                }
+            } else {
+                // unknown command
+                System.out.println(topBound + "Uh Oh! I'm sorry, but I don't know what that means." + bottomBound);
+            }
+        }
+    }
+    public static void readFromFile(String filePath, DukeList list) throws FileNotFoundException {
```

Comment:
There should be a line break to separate functions. JavaDoc for public functions need to be added also.